2021-11-18 12:58:23 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration average which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 12:58:23 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration nodered which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 12:58:23 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration lovelace_gen which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 12:58:23 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration weatherbit which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 12:58:23 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration powercalc which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 12:58:23 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration myenergi which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 12:58:23 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration alexa_media which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 12:58:23 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration alarmo which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 12:58:23 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration hpprinter which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 12:58:23 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 12:58:23 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration media_player_template which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 12:58:23 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration mail_and_packages which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 12:58:23 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration myq which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 12:58:23 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration auto_backup which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 12:58:23 WARNING (SyncWorker_4) [homeassistant.loader] We found a custom integration pirateweather which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 12:58:23 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration garbage_collection which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-18 12:58:43 WARNING (SyncWorker_2) [homeassistant.components.local_file.camera] Could not read camera mail_usps image from file: /custom_components/mail_and_packages/mail_none.gif
2021-11-18 12:58:44 WARNING (MainThread) [alexapy.alexalogin] No access token found; falling back to credential login instead of oauth.
2021-11-18 12:58:49 WARNING (MainThread) [homeassistant.setup] Setup of zone is taking over 10 seconds.
2021-11-18 12:58:51 WARNING (MainThread) [homeassistant.setup] Setup of webostv is taking over 10 seconds.
2021-11-18 12:58:54 WARNING (MainThread) [homeassistant.setup] Setup of automation is taking over 10 seconds.
2021-11-18 12:58:57 WARNING (MainThread) [alexapy.alexawebsocket] mac_dms not detected; websocket likely won't connect
2021-11-18 12:59:02 WARNING (MainThread) [aioesphomeapi.reconnect_logic] Can't connect to ESPHome API for 192.168.1.115 @ 192.168.1.115: Error connecting to ('192.168.1.115', 6053): [Errno 113] Connect call failed ('192.168.1.115', 6053)
2021-11-18 12:59:04 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input '<template TemplateState(<state sensor.ddwart_run_time_mins=7268.15; unit_of_measurement=mins, friendly_name=DDWart Run Time @ 2021-11-18T12:59:04.344877-05:00>)>' when rendering template '{% set time = (states.sensor.ddwart_run_time_mins | int) | int %} {% set minutes = ((time % 3600) / 60) | int %} {% set hours = ((time % 86400) / 3600) | int %} {% set days = (time / 86400) | int %}
{% if time == 0 %}
  Just restarted...
 {%- else -%}
  {%- if days > 0 -%}
    {{ days }}d
  {%- endif -%}
  {%- if hours > 0 -%}
    {%- if days > 0 -%}
      {{ ' ' }}
    {%- endif -%}
    {{ hours }}h
  {%- endif -%}
  {%- if minutes > 0 -%}
    {%- if days > 0 or hours > 0 -%}
      {{ ' ' }}
    {%- endif -%}
    {{ minutes }}m
  {%- endif -%}
{%- endif -%}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 12:59:04 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input '<template TemplateState(<state sensor.ddwart_run_time_mins=7268.15; unit_of_measurement=mins, friendly_name=DDWart Run Time @ 2021-11-18T12:59:04.344877-05:00>)>' when rendering template '{% set time = (states.sensor.ddwart_run_time_mins | int) | int %} {% set minutes = ((time % 3600) / 60) | int %} {% set hours = ((time % 86400) / 3600) | int %} {% set days = (time / 86400) | int %}
{% if time == 0 %}
  Just restarted...
 {%- else -%}
  {%- if days > 0 -%}
    {{ days }}d
  {%- endif -%}
  {%- if hours > 0 -%}
    {%- if days > 0 -%}
      {{ ' ' }}
    {%- endif -%}
    {{ hours }}h
  {%- endif -%}
  {%- if minutes > 0 -%}
    {%- if days > 0 or hours > 0 -%}
      {{ ' ' }}
    {%- endif -%}
    {{ minutes }}m
  {%- endif -%}
{%- endif -%}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 12:59:04 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input '<template TemplateState(<state sensor.netgear_r6400_v2_ewvisualarts_uptime=436089; unit_of_measurement=s, friendly_name=Netgear R6400 v2:EWVisualArts Uptime, icon=mdi:server-network @ 2021-11-18T12:59:03.065383-05:00>)>' when rendering template '{% set uptime = states.sensor.netgear_r6400_v2_ewvisualarts_uptime | int %} {% set days = (uptime / 86400) | int %} {%- if days > 0 -%}
  {{ days }} days, {{ (uptime - (days * 86400)) | int | timestamp_custom('%H:%M:%S', false) }}
{%- else -%}
  {{ uptime | int | timestamp_custom('%H:%M:%S', false) }}
{%- endif -%}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 12:59:04 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'int' got invalid input '<template TemplateState(<state sensor.netgear_r6400_v2_ewvisualarts_uptime=436089; unit_of_measurement=s, friendly_name=Netgear R6400 v2:EWVisualArts Uptime, icon=mdi:server-network @ 2021-11-18T12:59:03.065383-05:00>)>' when rendering template '{% set uptime = states.sensor.netgear_r6400_v2_ewvisualarts_uptime | int %} {% set days = (uptime / 86400) | int %} {%- if days > 0 -%}
  {{ days }} days, {{ (uptime - (days * 86400)) | int | timestamp_custom('%H:%M:%S', false) }}
{%- else -%}
  {{ uptime | int | timestamp_custom('%H:%M:%S', false) }}
{%- endif -%}' but no default was specified. Currently 'int' will return '0', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 12:59:04 ERROR (MainThread) [homeassistant.components.statistics.sensor] sensor.home_tank_usage: parsing error, expected number and received None
2021-11-18 12:59:13 ERROR (MainThread) [homeassistant.config_entries] Error unloading entry 192.168.1.24 for binary_sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 443, in async_unload
    result = await component.async_unload_entry(hass, self)  # type: ignore
  File "/usr/src/homeassistant/homeassistant/components/binary_sensor/__init__.py", line 162, in async_unload_entry
    return await component.async_unload_entry(entry)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 181, in async_unload_entry
    raise ValueError("Config entry was never loaded!")
ValueError: Config entry was never loaded!
2021-11-18 12:59:13 ERROR (MainThread) [homeassistant.config_entries] Error unloading entry 192.168.1.24 for camera
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 443, in async_unload
    result = await component.async_unload_entry(hass, self)  # type: ignore
  File "/usr/src/homeassistant/homeassistant/components/camera/__init__.py", line 368, in async_unload_entry
    return await component.async_unload_entry(entry)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 181, in async_unload_entry
    raise ValueError("Config entry was never loaded!")
ValueError: Config entry was never loaded!
2021-11-18 12:59:13 ERROR (MainThread) [homeassistant.config_entries] Error unloading entry 192.168.1.24 for sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 443, in async_unload
    result = await component.async_unload_entry(hass, self)  # type: ignore
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 134, in async_unload_entry
    return await component.async_unload_entry(entry)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 181, in async_unload_entry
    raise ValueError("Config entry was never loaded!")
ValueError: Config entry was never loaded!
2021-11-18 12:59:13 ERROR (MainThread) [homeassistant.config_entries] Error unloading entry 192.168.1.24 for switch
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 443, in async_unload
    result = await component.async_unload_entry(hass, self)  # type: ignore
  File "/usr/src/homeassistant/homeassistant/components/switch/__init__.py", line 85, in async_unload_entry
    return await component.async_unload_entry(entry)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 181, in async_unload_entry
    raise ValueError("Config entry was never loaded!")
ValueError: Config entry was never loaded!
2021-11-18 12:59:13 WARNING (MainThread) [homeassistant.helpers.template] Template warning: 'as_timestamp' got invalid input 'unavailable' when rendering template '{% set up_time =  as_timestamp(now()) - as_timestamp(states('sensor.ewvisualarts_last_boot')) %}
{% if up_time == 0 %}
  Just restarted...
{% else %}
  {% set minutes = (up_time // 60) | int %}
  {% set hours = (minutes // 60) %}
  {% set days = (hours // 24) %}
  {% set weeks = (days // 7) %}

  {% set minutes = (minutes % 60) %}
  {% set hours =  (hours % 24) %}
  {% set days = (days % 7) %}

  {% macro phrase(value, name) %}
            {%- set value = value %}
            {%- set end = 's' if value > 1 else '' %}
            {{- '{} {}{}'.format(value, name, end) if value | int > 0 else '' }}
  {%- endmacro %}

  {% set text = [ phrase(weeks, 'week'), phrase(days, 'day'), phrase(hours, 'hr'), phrase(minutes, 'min') ] | select('!=','') | list | join(', ') %}
  {% set last_comma = text.rfind(',') %}
  {% if last_comma != -1 %}
    {% set text = text[:last_comma] + '' + text[last_comma + 1:] %}
  {% endif %}

{{ text }}
{% endif %}' but no default was specified. Currently 'as_timestamp' will return 'None', however this template will fail to render in Home Assistant core 2022.1
2021-11-18 12:59:13 ERROR (MainThread) [homeassistant.components.template.template_entity] TemplateError('TypeError: unsupported operand type(s) for -: 'float' and 'NoneType'') while processing template 'Template("{% set up_time =  as_timestamp(now()) - as_timestamp(states('sensor.ewvisualarts_last_boot')) %}
{% if up_time == 0 %}
  Just restarted...
{% else %}
  {% set minutes = (up_time // 60) | int %}
  {% set hours = (minutes // 60) %}
  {% set days = (hours // 24) %}
  {% set weeks = (days // 7) %}

  {% set minutes = (minutes % 60) %}
  {% set hours =  (hours % 24) %}
  {% set days = (days % 7) %}

  {% macro phrase(value, name) %}
            {%- set value = value %}
            {%- set end = 's' if value > 1 else '' %}
            {{- '{} {}{}'.format(value, name, end) if value | int > 0 else '' }}
  {%- endmacro %}

  {% set text = [ phrase(weeks, 'week'), phrase(days, 'day'), phrase(hours, 'hr'), phrase(minutes, 'min') ] | select('!=','') | list | join(', ') %}
  {% set last_comma = text.rfind(',') %}
  {% if last_comma != -1 %}
    {% set text = text[:last_comma] + '' + text[last_comma + 1:] %}
  {% endif %}

{{ text }}
{% endif %}")' for attribute '_attr_native_value' in entity 'sensor.uptime_synology_nas'
2021-11-18 12:59:27 WARNING (MainThread) [homeassistant.components.media_player] Updating alexa_media media_player took longer than the scheduled update interval 0:00:10
2021-11-18 12:59:27 WARNING (MainThread) [homeassistant.helpers.entity] Update of media_player.livingroom_echo_dot is taking over 10 seconds
2021-11-18 13:23:21 ERROR (MainThread) [homeassistant.components.hassio.handler] Client error on /homeassistant/restart request Server disconnected
